### PR TITLE
operator: add additional info for operator (#2993)

### PR DESCRIPTION
**Pull request by ti-srebot**
cherry-pick #2993 to release-4.0

---

Signed-off-by: lhy1024 <admin@liudos.us>

### What problem does this PR solve?

When we check the log about create and finish operator, we don't know info about it. Especially, leader score and region score.

### Check List

Tests

- Unit test

### Release note

- add additional info for operator

### CODE REVIEW

**Review by disksing — APPROVED** [review]

File: server/schedulers/balance_test.go [hunk 1]
```diff
@@ -749,9 +751,9 @@ func (s *testBalanceRegionSchedulerSuite) TestReplacePendingRegion(c *C) {
 	sb, err := schedule.CreateScheduler(BalanceRegionType, oc, core.NewStorage(kv.NewMemoryKV()), schedule.ConfigSliceDecoder(BalanceRegionType, []string{"", ""}))
 	c.Assert(err, IsNil)
 
-	s.checkReplacePendingRegion(c, tc, opt, sb)
-	opt.EnablePlacementRules = true
-	s.checkReplacePendingRegion(c, tc, opt, sb)
+	s.checkReplacePendingRegion(c, tc, sb)
+	tc.SetEnablePlacementRules(true)
```
Comment (HunDunDM) [inline server/schedulers/balance_test.go:755]: Do not support in 4.0, please fix.

Comment (lhy1024) [reply server/schedulers/balance_test.go:755]: thanks!

**Review by HunDunDM — APPROVED** [review]
LGTM

**Pull request merged**